hydra-head.gemspec

```diff
@@ -18,14 +18,14 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'hydra-access-controls', version
   s.add_dependency 'hydra-core', version
-  s.add_dependency 'rails', '>= 5.2', '< 6.1'
+  s.add_dependency 'rails', '>= 5.2.4', '< 6.1'
 
   s.add_development_dependency 'coveralls'
-  s.add_development_dependency 'engine_cart', '~> 2.2'
+  s.add_development_dependency 'engine_cart', '~> 2.2.0'
```

Comment:
I know we tightened `engine_cart`'s versions when we were working on this, but I don't think it is necessary.  Could you try removing it and see if CI passes?